views/default/page/layouts/widgets/add_button.php

```diff
@@ -2,26 +2,30 @@
 /**
  * Button area for showing the add widgets panel
  */
+
+$href_options = [
+	'context' => elgg_get_context(),
+	'context_stack' => elgg_get_context_stack(),
+	'show_access' => elgg_extract('show_access', $vars, true),
+	'exact_match' => elgg_extract('exact_match', $vars, false),
+	'owner_guid' => elgg_extract('owner_guid', $vars, elgg_get_page_owner_guid()),
+];
+
+$href = elgg_normalize_url(elgg_http_add_url_query_elements('ajax/view/page/layouts/widgets/add_panel', $href_options));
+
 ?>
 <div class="elgg-widget-add-control">
 <?php
-	echo elgg_view('output/url', array(
-		'href' => '#',
+	echo elgg_view('output/url', [
+		'href' => 'javascript: void(0)',
 		'text' => elgg_echo('widgets:add'),
-		'class' => 'elgg-button elgg-button-action',
-		'rel' => 'toggle',
-		'is_trusted' => true,
-		'data-toggle-selector' => '#widgets-add-panel, .elgg-widget-add-control > a',
-		'data-toggle-slide' => 0,
-	));
-	echo elgg_view('output/url', array(
-		'href' => '#',
-		'text' => elgg_echo('widgets:panel:close'),
-		'class' => 'elgg-button elgg-button-action hidden',
-		'rel' => 'toggle',
-		'is_trusted' => true,
-		'data-toggle-selector' => '#widgets-add-panel, .elgg-widget-add-control > a',
-		'data-toggle-slide' => 0,
-	));
+		'class' => 'elgg-button elgg-button-action elgg-lightbox',
+		'data-colorbox-opts' => json_encode([
+			'href' => $href,
+			'maxWidth' => '900px',
+			'maxHeight' => '90%',
+		]),
+		'onclick' => 'require(["page/layouts/widgets/add_panel"])',
```

Comment:
Make this a menu item and pass with `items` to `elgg_view_menu('title:widgets')` ? Use menu class instead of `elgg-widgets-add-control`